src/scss/components/checkbox/_index.scss

```diff
@@ -3,8 +3,8 @@
 // ========== BASE ELEMENTS ==========
 hx-checkbox {
   display: inline-block;
-  height: 1rem;
-  width: 1rem;
+  height: $rax-spacing-400;
+  width: $rax-spacing-400;
```

Comment:
Please add a comment that the design token values are incorrect.

Reply:
Changes have been done, please review.